src/Tizen.NUI.Wearable/src/public/Popup.cs

```diff
@@ -0,0 +1,568 @@
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler OutsideClicked;
+
+        /// <summary>
+        /// Event when Post method is called. Customized action such as animation can be added.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler Posting;
+
+
+        /// <summary>
+        /// Event when Dismiss method is called. Customized action such as animation can be added.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler Dissmising;
+
+
+        /// <summary>
+        /// Get Popup style.
+        /// </summary>
+        /// <returns>The default popup style.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override ViewStyle GetViewStyle()
+        {
+            return new PopupStyle();
+        }
+
+        private void initialize()
+        {
+            popupStyle = Style as PopupStyle;
+            if (popupStyle == null)
+            {
+                throw new Exception("Popup style is null.");
+            }
+
+            instanceCnt++;
+            Name = "Popup_" + instanceCnt;
+            resourcePath = Tizen.Applications.Application.Current.DirectoryInfo.Resource;
+
+            layer = new Layer()
+            {
+                Name = "PopupLayer",
+            };
+
+            BackgroundColor = Color.Black;
+            Size = new Size(POPUP_WIDTH, POPUP_HEIGHT);
+            PositionUsesPivotPoint = true;
+            ParentOrigin = Tizen.NUI.ParentOrigin.Center;
+            PivotPoint = Tizen.NUI.PivotPoint.Center;
+            //CornerRadius = 0.5f; // this make me transparent!
+            layer.Add(this);
+
+            buttonList = new Dictionary<string, Button>();
+
+            title = new TextLabel()
+            {
+                Name = "Title",
+                Size = new Size(TITLE_WIDTH, TITLE_HEIGHT),
+                PointSize = TEXT_POINT_SIZE,
+                PositionUsesPivotPoint = true,
+                ParentOrigin = Tizen.NUI.ParentOrigin.TopCenter,
+                PivotPoint = Tizen.NUI.PivotPoint.TopCenter,
+                Position = new Position(0, TITLE_POSITION_Y),
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
+                TextColor = Color.Cyan,
+            };
+            Add(title);
+
+            LinearLayout scrollLayout = new LinearLayout();
+            scrollLayout.LinearOrientation = LinearLayout.Orientation.Vertical;
+
+            contentContainer = new View()
+            {
+                Layout = scrollLayout,
+                PositionUsesPivotPoint = true,
+                ParentOrigin = Tizen.NUI.ParentOrigin.TopCenter,
+                PivotPoint = Tizen.NUI.PivotPoint.TopCenter,
+            };
+
+            scroll = new ScrollableBase()
```

Comment:
If you want to divide popup into two area like below, you can use LinearLayout .
ㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡ
|⠀⠀⠀⠀⠀⠀title⠀⠀⠀⠀⠀⠀⠀⠀⠀|
ㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡ
|⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀|
|⠀⠀⠀ScrollableBase⠀⠀⠀⠀⠀|
|⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀|
ㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡ

If you set size of title and give weight 1 to ScrollableBase, the size of ScrollableBase is always popup size - title size.

Reply:
I tried to change it as using Layout, but it is still not shown. I will check continuously.

Reply:
Please don't forget using Layout later.